Makefile

```diff
@@ -12,6 +12,19 @@ aw-server:
 aw-webui:
 	make -C ./aw-webui build
 
+install-git-hooks:
+	@printf "make pre-commit\n" > .git/hooks/pre-commit
+	@printf "make pre-push\n" > .git/hooks/pre-push
+	@chmod +x .git/hooks/pre-commit .git/hooks/pre-push
+	@printf "Hooks installed\n"
```

Comment:
This print is pointless, it's never printed due to the @

Reply:
This is incorrect; `@` only supresses printing of the command, not stdout

Reply:
Aha, sorry